lib/core/engine.js

```diff
@@ -189,8 +184,8 @@ class Engine {
       generateCode(tasks[tasks.length - 1].contractsManager);
       self.events.once('outputDone', callback);
     });
-    const addToCargo = function (contractsManager) {
-      cargo.push({contractsManager});
+    const addToCargo = function () {
+      cargo.push({contractsManager: self.contractsManager});
```

Comment:
I don't think you need to pass the contract manager anymore. Looks like generateCode doesn't need it anymore.